committee/wg-governance/wg-governance.md

```diff
@@ -1 +1,57 @@
-TBD: will be finished after [wg-governance-zh_CN.md] is merged.
\ No newline at end of file
+# TiKV Working Group Governance
+
+## What is Working Group?
+
+Contributors are totally able to just work on goals together without it being a WG or formal. We're not 'allowing' it, this is just a formal recognition that this will happen and that WGs can empower and enhance these cooperations.
+
+Due to the Working Group usually solves a problem that spans the scope of responsibility of multiple Special Interest Groups (SIGs), so it usually consists of members from different SIGs or some other community contributors. The Working Group usually dismisses after the problem is solved. And the results or members in the Working Group can be accepted by the SIG after. For more detailed procedures of creating or dismiss, see [WG Lifecycle](#wg-lifecycle).
```

Comment:
```suggestion
A Working Group usually solves a problem that spans the scope of responsibility of multiple Special Interest Groups (SIGs), so it usually consists of members from different SIGs or some other community contributors. The Working Group usually dissolves after the problem is solved. For more detailed procedures of creating or dismiss, see [WG Lifecycle](#wg-lifecycle).
```

Removed a sentence that seems to imply contributors cannot be in both a SIG and a WG